Cargo.toml

```diff
@@ -8,6 +8,8 @@ repository = "https://github.com/rust-lang/docs.rs"
 build = "build.rs"
 edition = "2018"
 
+[workspace]
```

Comment:
What's the purpose of the empty workspace?

Reply:
https://doc.rust-lang.org/cargo/reference/workspaces.html#workspaces

> A workspace can be created by adding a [workspace] section to Cargo.toml. This can be added to a Cargo.toml that already defines a [package], in which case the package is the root package of the workspace. The workspace root is the directory where the workspace's Cargo.toml is located.

Reply:
Well yah, but why a workspace at all since this is essentially just a dependency

Reply:
See @pietroalbini's comments above - without a workspace the tests won't be run in CI. https://github.com/rust-lang/docs.rs/pull/1000#discussion_r475421475